### PR TITLE
Add Godot bomber demo

### PR DESCRIPTION
**Pull request by AngelOnFira**

### CODE REVIEW

**Comment by AngelOnFira** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/rivet-gg/site/70?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #23** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/23?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #24** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/24?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #68** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/68?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #69** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/69?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #70** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/70?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rivet-gg/site/70?utm_source=stack-comment).

**Comment by AngelOnFira** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/rivet-gg/site/70?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #23** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/23?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #24** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/24?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #68** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/68?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #69** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/69?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #70** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/70?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rivet-gg/site/70?utm_source=stack-comment).

File: src/pages/learn/godot/index.mdx [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import { LearnHome } from '@/components/LearnHome';
 
+import tutorialTanks from 'src/pages/learn/html5/tutorials/previews/tanks-canvas-socketio.png';
```
Comment (AngelOnFira) [inline src/pages/learn/godot/index.mdx:3]: @NicholasKissel where can I find the Bomber Demo box art?

**Comment by AngelOnFira** [comment]
@NicholasKissel  
> Thoughts on adding a design language around tips vs where you get stuck for the drop downs -- either via color or emojis.  Such as 💡 for "What did this do?💡" and ⚠️ "Rivet Panel Missing? ⚠️"

Ya, this would be super solid. I imagine it could be built into a component somehow, so instead of doing an accordion, I would do either `<Information>` or `<Warning>`.

Do you also mean inlining more instead of having everything as dropdowns?

**Review by NathanFlurry — DISMISSED** [review]
only major problem i see: the default regions changed, see https://sandbox.rivet.game/matchmaker.html

otherwise i'm going to yolo merge and go review at the end of this week.

**Comment by AngelOnFira** [comment]
ty for the catch, resolved this 👍

**Pull request merged**